external/llvm-project/mlir/lib/Dialect/Tosa/Transforms/TosaPartition.cpp

```diff
@@ -418,18 +491,35 @@ class TosaPartitionPass : public TosaPartitionBase<TosaPartitionPass> {
   }
 
   void runOnOperation() override {
+    // Must set config here, because at pass-construction time the options
+    // haven't been parsed yet.
+    if (!config) {
+      if (anchorOps.hasValue() || attributeName.hasValue() ||
```

Comment:
If *any* sub-option is set, go this way.  Option defaults make sure it behaves rationally.